packages/app/src/pages/components/Login.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react'
+import {redirectTo, getCookieValue, setCookie} from '../utils'
+import * as styles from '../index.module.sass'
+
+const HOME = 'http://localhost:8000'

```

Comment:
Please use environment variables with self-descriptive names. For example: `URL_HOME` and `URL_OAUTH_LOGIN`.


[See more here.](https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/#accessing-environment-variables-in-the-browser)